src/Examine/LuceneEngine/Providers/LuceneIndex.cs

```diff
@@ -43,8 +44,22 @@ public class LuceneIndex : BaseIndexProvider, IDisposable, IIndexStats
             Analyzer analyzer = null,
             IValueSetValidator validator = null,
             IReadOnlyDictionary<string, IFieldValueTypeFactory> indexValueTypesFactory = null)
-            : this(name, luceneDirectory, DefaultQueueCapacity, fieldDefinitions, analyzer, validator, indexValueTypesFactory)
+            : this(name, luceneDirectory, DefaultQueueCapacity, fieldDefinitions, analyzer, validator,
+                indexValueTypesFactory)
         {
+            if (luceneDirectory is ExamineDirectory dir)
+            {
+                isReadonly = dir.IsReadOnly;
+                if (dir.WriteLogToLocalDirectory)
```

Comment:
That shouldn't be managed by LuceneIndex.

Reply:
done